marlowe-playground-client/src/SimulationPage/State.purs

```diff
@@ -294,16 +291,21 @@ editorSetValue contents = void $ query _simulatorEditorSlot unit (Monaco.SetText
 editorGetValue :: forall state action msg m. HalogenM state action ChildSlots msg m (Maybe String)
 editorGetValue = query _simulatorEditorSlot unit (Monaco.GetText identity)
 
-updateContractInEditor ::
+updateOracleAndContractEditor ::
   forall m.
   MonadAff m =>
   MonadAsk Env m =>
   HalogenM State Action ChildSlots Void m Unit
-updateContractInEditor = do
-  executionState <- use (_currentMarloweState <<< _executionState)
-  editorSetValue
-    ( case executionState of
-        SimulationRunning runningState -> show $ genericPretty runningState.contract
-        SimulationNotStarted notStartedState -> maybe "No contract" (show <<< genericPretty) notStartedState.extendedContract -- This "No contract" should never happen if we get valid contracts from editors
-    )
+updateOracleAndContractEditor = do
+  mContract <- peruse _currentContract
+  -- Update the decorations around the current part of the running contract
+  case (getLocation <$> mContract) of
```

Comment:
I think this parenthesis is unnecessary

Reply:
will do in following pr so we don't wait on hydra